source/a_releasenotes.rst

```diff
@@ -8,95 +8,87 @@ Release 3Di 8 March 2021
-=============================
-
-Long crested weirs: The formulation of the long crested weir has been replaced by a new one. This new version is based on the law of Bernouilli instead of an alternative implementation of the advective terms as a 1D element. The flow over the weir is an  accurate computation of the flow under ideal circumstances, but the new formulation does not require an extra computational node and has proven to be more stable under varying flow conditions.
-
-Short crested weirs:
-Flow over a weir knows three different stages: sub, supercritical & a transtion of critical flow.  
-The type of flow is determined by the downstream waterlevel.
-In case of a water level downstream that has no influence on the flow over the weir it is called supercritical. This part remains the same. What has changed is the formula for subcritical condition and then only in case of a negative flow. 
-
-Next to that the time dependency of the flow has been adjusted. This will have no effect on stationairy flow, but can cause a little more or less traagheid in your system. 
+Bug fixes in calculation core
+========================
 
 In short the following fixes are included in the calculation core:
-
-- Fixes for long crested weir; New routine that does not request an extra computational node → does this mean that users can do stuff differently? 
+- Fix for long crested weir; new routine that does not request an extra computational node. 
 - Fix for short crested weir; Fix to determine super- from sub-critical regime. 
-- Fix for weirs for negative subcritical flows → isn;t this the same as the previous one? 
+- Fix for weirs for negative subcritical flows 
 - Fix for assigning boundary locations at North and East sides, in case computational cells stretch over original DEM raster, due to refinements. 
 - Fix for 1D coordinates in netcdf file: The z-coordinates of the boundary points, are now set correctly in the netcdf
-- Fix for initial conditions in netcdf file: In case of 1D-2D models, some variables, like the wet-surface areas of a computational node, the wrong value was written in the results netcdf. 
-- Fix for interception, need for interception is verified before going over all 2D nodes. (Answers should be the same, but computational time much faster.) 
+- Fix for initial conditions in netcdf file: In case of 1D-2D models, some variables, like the wet-surface areas of a computational node, the wrong value was written in the results netcdf at the start of the simulation. 
+
+Long crested weirs: The formulation of the long crested weir has been replaced by a new one. This new version is based on the law of Bernouilli instead of an alternative implementation of the advective terms for a regular 1D element. The flow over the weir is an accurate computation of the flow under ideal circumstances, but the new formulation does not require an extra computational node and has proven to be more stable under varying flow conditions.
```

Comment:
Ik weet niet of deze twee teksten hierbij horen, bij de andere bug fixen is ook geen uitleg. Kijk maar